libs/sdk-ui-filters/src/AttributeFilter/hooks/useAttributeFilterController.ts

```diff
@@ -780,6 +780,8 @@ function useCallbacks(
         }
     }, [handler, supportsKeepingDependentFiltersSelection, supportsShowingFilteredElements]);
 
+    useReportMigratedFilter(handler, onApply);
```

Comment:
I would expect this to reexecute the dashboard insights on init. Which might be perf/resource problem.

Did not test it though.

If not, can you please explain me why not?

Reply:
Yes, it will cause double execution. This is the cost to get the features that are affected working.

The code is triggered only on old dashboards with filters that use alt labels. You can edit the dashboard, change something in the filter, change it back, save the dashboard, and the code will stop triggering as it is no longer needed. 

I'm afraid we will have to do metadata migration of all the affected dashboards down the road which will fix this permanently.